ui/jobs/buff_tracker.ts

```diff
@@ -428,33 +445,41 @@ export class BuffTracker {
       standardFinish: {
         gainEffect: EffectId.StandardFinish,
         loseEffect: EffectId.StandardFinish,
-        durationSeconds: 60,
+        useEffectDuration: true,
         icon: '../../resources/ffxiv/status/standard-finish.png',
         // Green.
         borderColor: '#32CD32',
         sortKey: 6,
       },
       technicalFinish: {
+        // Sorry, but 0-3 step Technical Finish will not trigger this if it misses you.
+        // and I think it meaningless to trace a poor DNC
+        // who can perform 0-3 step Technical Finish and miss you at the same time.
```

Comment:
Haha, that's fair.  I think we could support an array of abilities for `cooldownAbility` (or an array of effects for `gainEffect` and combine embolden/emboldenSelf), and we already have all the ability ids for the other technical finishes.  Just saying we *couuuuld* do it, but agree that "I don't think we should bother" is a fine answer.  :)

Reply:
Sounds great.
By the way, `Technical Step` cooldown at start dancing, but take effect on finish, the time between this is variable so it may also not accurate.

Reply:
Ah, that's fair! Maybe it's worth shortening the cooldown by the "usual" time it takes to complete?

Reply:
I think it not necessary? If DNC always dance on time and make every step right, cooldown of buff is exactly 120, for cooldown is triggered on finish now.

Reply:
Oh ok, then it's just if they add extra delay then it might not be accurate.

Reply:
Changed to array to handle all kinds of finish.
Though cooldown may still not be accurate if dancing some extra step or less.